C-Plus-Plus/other/count_of_set_bit_using_lookup_tech.cpp

```diff
@@ -0,0 +1,47 @@
+/*A CPP implementation of count of set bits in an integer using a LookUp 
+table technique.
+
+Piyush Kumar
+
+ */
+
+#include <bits/stdc++.h>
+#include <iostream>
+#include <algorithm>
+#include <vector>
+using namespace std;
+
+int main()
+{
+    int arr[] = {0, 1, 1, 2, 1, 2, 2, 3, 1, 2, 2, 3, 2, 3, 3, 4}; // An array containg count of set bits from 0 - 15
```

Comment:
Please write the generalized case not only limited to fix value till 15. Set bits may exceed from 4 if you use a bigger number